app/src/services/sync.js

```diff
@@ -81,7 +127,8 @@ const service = {
             response[0].versions.find(v => v.id === version.id).metadata = metadata;
           }
         }
-        log.verbose(`Finished syncing  ${path} in bucket ${bucketId}`, { function: 'syncJob', result: response });
+
+        log.verbose(`Finished syncing ${path} in bucket ${bucketId}`, { function: 'syncJob', result: response });
         return response;
```

Comment:
we don't actually expose this response to the api user.. but it seems like a nice thing to have (a nested array of object>version?meta/tags) for logging.